src/client/src/rt-components/route-wrapper/RouteWrapper.tsx

```diff
@@ -31,27 +29,12 @@ interface RouteWrapperProps {
   title?: string | SymbolParamObject
 }
 
-// TODO Move to openfin-platform
-//@ts-ignore
-const isChildView = window.fin && window.fin.me && window.fin.me.isView
-
 const RouteWrapper: React.FC<RouteWrapperProps> = props => {
-  const { children, windowType = 'main', title } = props
-  const [fromLauncher, setFromLauncher] = useState<boolean>(false)
+  const { children, windowType = 'main' } = props
   const platform = usePlatform()
   const theme = useTheme()
 
-  const { PlatformHeader, PlatformFooter, PlatformControls, PlatformRoute, window } = platform
-
-  useEffect(() => {
-    isParentAppOpenfinLauncher()
```

Comment:
@stewart-adaptive this particular bit - `isParentAppOpenfinLauncher` - was added, I think, for the OpenFin windows that you can launch by clicking on the buttons like "Open Live Rates", "Open Trades" and "Open Tile".

In those cases the launcher is directly launching the routes themselves, disconnected from the main app (i.e. they have a "close" button, not a "pop back in" button.

Can you confirm that with this change that functionality is still working?

![image](https://user-images.githubusercontent.com/3295115/97599111-8db90580-19ff-11eb-93b8-61cadbb5ae27.png)

Reply:
you are right, Ill look into a fix

Reply:
Updated and fixed close button issue, ![image](https://user-images.githubusercontent.com/30235606/97696297-48e3ac00-1a9d-11eb-8e35-0f744dd40031.png)